README.md

```diff
@@ -19,12 +19,25 @@ Clone the repository
 git clone https://github.com/greypants/gulp-rails-pipeline.git
 ```
 
-Enter into the directory and run bundle install
+Enter into the directory and run bundle install.
 ```
 cd gulp-rails-pipeline
 bundle install
 ```
 
+If you are on mac you may be required to run the following to [install PostgreSQL libraries successfully](http://stackoverflow.com/a/28206009/1376627).
```

Comment:
I didn't include PostgreSQL as a dependency here... this bit might be distracting/confusing to anyone not using it.